src/OpenTelemetry.Contrib.Instrumentation.Runtime/RuntimeMetricsOptionsExtensions.cs

```diff
@@ -0,0 +1,94 @@
+﻿// <copyright file="RuntimeMetricsOptionsExtensions.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using OpenTelemetry.Contrib.Instrumentation.Runtime;
+
+namespace OpenTelemetry.Metrics
+{
+    /// <summary>
+    /// Extensions to configure the runtime metrics options.
+    /// </summary>
+    public static class RuntimeMetricsOptionsExtensions
```

Comment:
I think we don't need to have this (atleast in an initial release).As `RuntimeMetricsOptions` already has public properties to enable each category of metrics.

Reply:
> the general idea - start with the absolute minimum public API. We can always add more public APIs based on feedbacks from initial release.

+1 👍